src/frontend/apps/standalone_site/src/features/PortabilityRequests/components/PortabilityRequests.tsx

```diff
@@ -0,0 +1,173 @@
+import { Box, Button, Heading, Text } from 'grommet';
+import { Maybe } from 'lib-common';
+import { Spinner } from 'lib-components';
+import { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { ReactComponent as CheckListIcon } from 'assets/svg/iko_checklistsvg.svg';
+import { WhiteCard } from 'components/Cards';
+import { SortableTable } from 'components/SortableTable';
+
+import { usePortabilityRequests } from '../api/usePortabilityRequests';
+import { useLtiUserAssociationJwtQueryParam } from '../hooks/useLtiUserAssociationJwtQueryParam';
+
+import { ItemTableRow } from './ItemTableRow';
+
+const messages = defineMessages({
+  title: {
+    defaultMessage: 'Portability requests',
+    description: 'PortabilityRequests title',
+    id: 'features.PortabilityRequests.title',
+  },
+  noPortabilityRequest: {
+    defaultMessage: 'You have no portability requests yet.',
+    description:
+      'Message displayed when there is no portability request to display',
+    id: 'features.PortabilityRequests.noPortabilityRequest',
+  },
+  tableTitle: {
+    defaultMessage:
+      '{item_count, plural, =0 {no portability request} one {# portability request} other {# portability requests}}',
+    description: 'Portability requests table title.',
+    id: 'features.PortabilityRequests.tableTitle',
+  },
+  error: {
+    defaultMessage: 'An error occurred, please try again later.',
+    description: 'Error message on loading portability requests',
+    id: 'features.PortabilityRequests.error',
+  },
+  retry: {
+    defaultMessage: 'Retry',
+    description: 'Retry button title',
+    id: 'features.PortabilityRequests.retry',
+  },
+  sortByAscendingCreationDate: {
+    defaultMessage: 'Creation date',
+    description: 'Sort portability requests by ascending creation date',
+    id: 'features.PortabilityRequests.sortByAscendingCreationDate',
+  },
+  sortByDescendingCreationDate: {
+    defaultMessage: 'Creation date (reversed)',
+    description: 'Sort portability requests by descending creation date',
+    id: 'features.PortabilityRequests.sortByDescendingCreationDate',
+  },
+});
+
+interface PortabilityRequestsProps {
+  state?: Maybe<string>;
+  for_playlist_id?: Maybe<string>;
+}
+
+export const PortabilityRequests = ({
+  state,
+  for_playlist_id,
+}: PortabilityRequestsProps) => {
+  const intl = useIntl();
+
+  const sorts = [
+    {
+      id: '-created_on',
+      label: intl.formatMessage(messages.sortByDescendingCreationDate),
+    },
+    {
+      id: 'created_on',
+      label: intl.formatMessage(messages.sortByAscendingCreationDate),
+    },
+  ];
+  const pageSize = 10;
```

Comment:
We have a global ITEM_PER_PAGE in src/conf/global.ts, not sure that it fits your case though.